src/microbe_stage/ProcessStatistics.cs

```diff
@@ -26,15 +36,15 @@ public void RemoveUnused()
         }
     }
 
-    public SingleProcessStatistics GetAndMarkUsed(TweakedProcess forProcess)
+    public SingleProcessStatistics GetAndMarkUsed(BioProcess forProcess)
     {
         if (Processes.TryGetValue(forProcess, out var entry))
         {
             entry.Used = true;
             return entry;
         }
 
-        entry = new SingleProcessStatistics(forProcess.Process);
+        entry = new SingleProcessStatistics(forProcess);
```

Comment:
Ah, good spot simplifying this.

Reply:
Well this was not really about simplifying. I initially designed this to take in a TweakedProcess in case there was a situation where multiple TweakedProcesses might exist for the same base process and their statistics should be able to exist separately. I've now decided that that is a very confusing and bad thing to allow, so I adjusted the object used as key to store the speed information.